main.rb

```diff
@@ -36,17 +40,24 @@ def handle_option(option)
     when 2
       @my_app.list_people
     when 3
-      handle_create_person
+      @person_handler.handle_create_person
     when 4
-      handle_create_book
+      @books_handler.handle_create_book
     when 5
-      handle_create_rental
+      @rentals_handler.handle_create_rental
     when 6
-      handle_list_rentals
+      @rentals_handler.handle_list_rentals
     else
       puts 'Invalid option'
     end
   end
+end
+
+# person handler
```

Comment:
- [ ] According to the project requirements, this should be the end of the `main.rb` file. Kindly the rest of the code (PersonHandler, Rentals, and BooksaHandler classes) into a seperate `app.rb` file and import it to main. Please also remember to check that your app still works as expected after making the changes.

Reply:
@NduatiKagiri Thanks for the review the requested changes will be implemented. :slightly_smiling_face: